### ISSUE TITLE
restore Transactions

### ISSUE THREAD

**Issue by sbenodiz** [robotmedia/AndroidBillingLibrary#64]
The problem start when I do wipe to the data of the app ,after that My app don't get the old transactions even when I call to :BillingController.restoreTransactions(this);

**Comment by jonathanyee** [comment]
Yes, I believe restoreTransactions only restores items from the database.  Would be nice to be able to restore purchases from the play store.

**Comment by hpique** [comment]
@sbenodiz I'll give this a look. In which devices have you reproduced this behavior?

@raptrex No, it restores transactions from the Play Store.
